### DISCUSSION TITLE
Blog-Doc: The Simplest Node.js CMS & SSG

### DISCUSSION THREAD

**Discussion by LebCit** [eta-dev/eta#248] [Show and tell]
Hello,

I've just rebuilt [Blog-Doc](https://github.com/LebCit/blog-doc) from ground-up using Hono, Eta and Marked.
You can read more about the new version of Blog-Doc in the following article: [From 145 to 7 💪 ](https://lebcit.github.io/posts/from-145-to-7/)

A big thanks to all Eta developers for their time and efforts, bringing a 0 dependency embedded JS template engine.

**Comment by bgub** [comment]
So cool! Thanks for sharing 🚀
